core/src/main/resources/reference.conf

```diff
@@ -184,6 +184,8 @@ backend {
         #concurrent-job-limit = 5
 
         run-in-background = true
+        # Configures any logic which should run after the command completes
```

Comment:
I don't know that it has to be single line necessarily, but I like the rest of that suggestion.